packages/amplify-graphql-api-construct-tests/src/__tests__/backends/configurable-stack/app.ts

```diff
@@ -33,9 +37,12 @@ const combineTestDefinitionsInDirectory = (directory: string): IAmplifyGraphqlDe
   return AmplifyGraphqlDefinition.combine(definitions);
 };
 
-// Keeping this short so we don't bump up against resource length limits (e.g., 140 characters for lambda layers).
+const projRoot = path.normalize(path.join(__dirname, '..'));
+
+// Read the prefix to use when naming stack assets. Keep this short so you don't bump up against resource length limits (e.g., 140
+// characters for lambda layers).
```

Comment:
Since these prefixes are currently hand-written, 
- how do we know what's the max prefix length that won't cross 140?
- can we trim it to say 100 chars long which we know will succeed before deploying? I don't think we will need those many chars since these are essentially like test IDs.

Reply:
The prefix must be way shorter than 140 -- the example is the lambda resource that broke last time I ran into the length limit. In an upcoming PR I'll truncate to <= 20 chars